packages/frontend/tests/features/bs-student-onboarding-journey.feature.txt

```diff
@@ -0,0 +1,9 @@
+Feature: The student onboarding journey
+
```

Comment:
I think the file name shouldn't include the environment or agent-specific descriptions. Ideally, all tests should run on the agent environment that was specified by the command that is running them.

Just noticed that this is a `.txt` file. Shouldn't that be a `.feature`?? Or is this file needed to be deleted along with the `google.js` ?

Reply:
There was red in the test stage of ci because we couldn't write a test scenario not yet.